gym/spaces/dict.py

```diff
@@ -1,15 +1,23 @@
 """Implementation of a space that represents the cartesian product of other spaces as a dictionary."""
+import sys
 from collections import OrderedDict
 from collections.abc import Mapping, Sequence
 from typing import Any
 from typing import Dict as TypingDict
-from typing import Optional, Union
+from typing import List, Optional
+from typing import Sequence as TypingSequence
+from typing import Tuple, Union
 
 import numpy as np
 
 from gym.spaces.space import Space
 from gym.utils import seeding
 
+if sys.version_info >= (3, 8):
+    from typing import OrderedDict as TypingOrderedDict
```

Comment:
Why do we need this? Since 3.7, regular dict is basically an ordered dict. And even then, OrderedDict is a subtype of Dict, so `Union[Dict, OrderedDict]` is the exact same thing as `Dict`

Reply:
yes, but we still support python 3.6... where this is not true sadly so I think we need to keep this for now or ignore it

Reply:
Will do